lib/slack.rb

```diff
@@ -1,11 +1,65 @@
-#!/usr/bin/env ruby
+require 'dotenv'
+require 'table_print'
+require_relative 'workspace'
 
 def main
+  Dotenv.load
+
   puts "Welcome to the Ada Slack CLI!"
   workspace = Workspace.new
+  user_input = ""
 
   # TODO project
 
+  until user_input == "quit"
+    puts "\nWhat would you like to do? Please select one options: \n1. list users\n2. list channels\n3. select user\n4. select channel\n5. details\n6. send message\n7. quit"
+    user_input = gets.chomp.downcase
+    puts
+    case user_input
+    when "1", "list users"
+      tp workspace.users, :real_name, :user_name, :slack_id
+    when "2", "list channels"
+      tp workspace.channels, :name, :topic, :member_count, :slack_id
+    when "3", "select user"
```

Comment:
This could be a touch more readable!
```suggestion
    when "1", "list users"
      tp workspace.users, :real_name, :user_name, :slack_id
      
    when "2", "list channels"
      tp workspace.channels, :name, :topic, :member_count, :slack_id
      
    when "3", "select user"
```

Etc!

Reply:
sorry I'm confused, I didn't understand what you meant by Etc